tls/s2n_fingerprint_ja4.c

```diff
@@ -64,18 +64,16 @@ const char *s2n_ja4_version_strings[] = {
     [0x0302] = "11",
     [0x0301] = "10",
     /**
-     *= https://raw.githubusercontent.com/FoxIO-LLC/ja4/v0.18.2/technical_details/JA4.md#tls-version
+     *= https://raw.githubusercontent.com/FoxIO-LLC/ja4/df3c067/technical_details/JA4.md#tls-and-dtls-version
      *# 0x0300 = SSL 3.0 = “s3”
-     *# 0x0200 = SSL 2.0 = “s2”
-     *# 0x0100 = SSL 1.0 = “s1”
+     *# 0x0002 = SSL 2.0 = “s2”
      */
     [0x0300] = "s3",
     [0x0200] = "s2",
```

Comment:
I'm not sure if it matters since we don't support sslv2, but should this be 0x0002?
```suggestion
    [0x0002] = "s2",
```

Reply:
Oh nice catch. I love it when compliance comments make non-compliance easier to notice :)